sdk/monitor/query/azmetrics/README.md

```diff
@@ -73,6 +76,9 @@ comments.
 [azure_sub]: https://azure.microsoft.com/free/
 [cloud_documentation]: https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/azcore/cloud
 [default_cred_ref]: https://github.com/Azure/azure-sdk-for-go/tree/main/sdk/azidentity#defaultazurecredential
+[ingestion/azlogs]: https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/monitor/ingestion/azlogs
```

Comment:
I don't know of one that wouldn't, but I'm afraid some markdown parser won't allow `/` in a link label
```suggestion
[ingestion_azlogs]: https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/monitor/ingestion/azlogs
```

Reply:
fixed in [63dda26](https://github.com/Azure/azure-sdk-for-go/pull/22729/commits/63dda26dd58e2aa4a3d108fc27e4cbfa30bfd77c)